instance/models/openedx_appserver.py

```diff
@@ -603,88 +618,116 @@ def provision(self):
 
         Returns True on success or False on failure
         """
-        # pylint: disable=cyclic-import, useless-suppression
-        from instance.models.openedx_deployment import OpenEdXDeployment
+        try:
+            return self._do_provision()
+        except ProvisioningError as pe:
+            self.logger.exception(pe.message)
+            self.provision_failed_email(pe.message, pe.log)
```

Comment:
Since there will be no `log` in `ProvisioningError`, we can skip sending it. This will simplify and reduce changes in some tests as well.
```suggestion
            self.provision_failed_email(pe.message)
```

Reply:
Resolving this based on [response here](https://github.com/open-craft/opencraft/pull/821#discussion_r663508376)